backend/app/rest/tag_routes.py

```diff
@@ -0,0 +1,32 @@
+from flask import Blueprint, current_app, jsonify, request
```

Comment:
@owen-sellner mentioned this on @braydonwang 's GET PR, but just to keep naming conventions consistent could you rename this to `tags_routes.py`